app/views/customers/_customer.html.erb

```diff
@@ -9,6 +9,7 @@
     </div>
   </td>
   <td><span class="fw-normal text-wrap"><%= customer.company %></span></td>
+  <td><span class="fw-normal text-wrap"><%= customer.bio %></span></td>
```

Comment:
I'm not looking to add a `bio` column to the standard Beast Mode app. This is going to have to come out.